src/components/pages/Search/Search.tsx

```diff
@@ -0,0 +1,100 @@
+import RoutePointer from '../../RoutePointer/RoutePointer';
+import Alphabet from '../../Alphabet/Alphabet';
+import InputField from '../../InputField/InputField';
+import SearchGrid from '../../common/SearchGrid';
+import ITeacherCard from '../../I-TeacherCard/I-TeacherCard';
+
+import {Location, useLocation} from 'react-router-dom';
```

Comment:
On the other hand, we have  `import { RouterProvider } from 'react-router-dom';` let's provide the same behavior with spaces between brackets `{ }`

I would choose spaces `import { Location, useLocation } from 'react-router-dom';`. What do you think?